File: BMI v.3.0/app/src/main/java/pl/arisa/bmicalculator/ui/chart/ChartFragment.java
package pl.arisa.bmicalculator.ui.chart;

import android.content.Context;
import android.os.Bundle;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.webkit.WebView;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

import pl.arisa.bmicalculator.R;
import pl.arisa.bmicalculator.ui.bmi.BmiFragment;


public class ChartFragment extends Fragment {
    WebView wb_chart;

    @Override
    public View onCreateView(@NonNull LayoutInflater inflater,
                             ViewGroup container, Bundle savedInstanceState) {

        View root = inflater.inflate(R.layout.fragment_chart, container, false);

        return root;
    }

    @Override
    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);

        wb_chart = (WebView) view.findViewById(R.id.webview_chart);
        wb_chart.loadUrl("https://ourworldindata.org/obesity");
    }
}

File: BMI v.4.0/app/src/main/java/pl/arisa/bmicalculator/ui/chart/ChartFragment.kt
package pl.arisa.bmicalculator.ui.chart

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.webkit.WebView
import androidx.fragment.app.Fragment
import pl.arisa.bmicalculator.R

class ChartFragment : Fragment() {
    var wb_chart: WebView? = null
    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?, savedInstanceState: Bundle?
    ): View? {
        return inflater.inflate(R.layout.fragment_chart, container, false)
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        wb_chart = view.findViewById<View>(R.id.webview_chart) as WebView
        wb_chart!!.loadUrl("https://ourworldindata.org/obesity")
    }
}